### PR TITLE
INGM-433 Example script to check servo configurations

### PR DESCRIPTION
**Pull request by mjedieckmann**
### Description

Fixes # (https://novantamotion.atlassian.net/browse/INGM-433)

### Type of change

- [x] Add check_configuration function to ingeniamotion
- [x] Provide example file on how to use it to check the configuration of a given list of servos

### Documentation

Please update the documentation.

- [x] Update docstrings of every function, method or class that change.
- [x] USe [type hints](https://docs.python.org/3/library/typing.html) for every function and argument.
- [x] Build documentation locally to verify changes.
- [x] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting

- [x] Use ruff to format the code

### CODE REVIEW

File: examples/check_configurations_example.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    print("Starting script to check configurations..")
+    print("Checking for each servo if their current configuration matches the given file.")
+    # The folder where you placed the dictionary and configuration - files for the servos
+    # you defined in SERVOS_TO_TEST.
+    resources_folder = "examples/resources/"
+    mc = MotionController()
+
+    for servo_to_test in SERVOS_TO_TEST:
+        ip = servo_to_test["ip"]
+        config = resources_folder + servo_to_test["config"]
```
Comment (pzinemanas) [inline examples/check_configurations_example.py:39]: Use `os.path.join` or similar to join paths,

**Review by pzinemanas — CHANGES_REQUESTED** [review]
Please add a test in test_examples.py.

File: ingeniamotion/configuration.py [hunk 2]
```diff
@@ -161,6 +161,35 @@ def default_brake(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS) ->
...
+        Raises:
+            FileNotFoundError: If configuration file does not exist.
+            ValueError: If a configuration file from a subnode different from 0
+                is attempted to be loaded to subnode 0.
+            ValueError: If an invalid subnode is provided.
+            ILConfigurationError: If the configuration file differs from the drive state.
+
+        """
+        if not path.isfile(config_path):
+            raise FileNotFoundError("{} file does not exist!".format(config_path))
```
Comment (MartinALL91) [inline ingeniamotion/configuration.py:186]: Use an f-string

File: ingeniamotion/configuration.py [hunk 3]
```diff
@@ -161,6 +161,35 @@ def default_brake(self, servo: str = DEFAULT_SERVO, axis: int = DEFAULT_AXIS) ->
         """
         self.disable_brake_override(servo, axis)
 
+    def check_configuration(
+        self, config_path: str, axis: Optional[int] = None, servo: str = DEFAULT_SERVO
+    ) -> None:
```
Comment (MartinALL91) [inline ingeniamotion/configuration.py:166]: I think It would be best if the method return a boolean.

Comment (mjedieckmann) [reply ingeniamotion/configuration.py:166]: The issue specified that the ingenialink function should throw an exception when the check fails:
image
I could change this function to catch the error and return a boolean, but I'm not sure if this is what we need.

**Review by GullinBustin — APPROVED** [review]

**Pull request merged**